main.go

```diff
@@ -12,6 +12,7 @@ import (
 
 const httpBaseUrlDefault = "http://127.0.0.1/tftp"
 const tftpTimeoutDefault = 5 * time.Second
+const tftpBindAddrDefault = "0.0.0.0:69"
```

Comment:
Please don't change the default - leave it as `:69`. As I explained in #3, that will listen on both IPv6 and IPv4, vs what you are proposing, which would only listen on IPv4.